src/components/covariant-menu/covariant-menu.tsx

```diff
@@ -53,14 +54,58 @@ export const CovariantMenu = ({
     nudgeFunction: nudgeDisplayMode,
   }
 
+  const {
+    styles,
+    menuSlideTranslation,
+    menuScale,
+    menuBackgroundColor,
+    labelOpacity,
+    labelCounterScale,
+  } = getMenuStylesAndAnimationVals(hideMenu, hideLabel, 'LEFT')
+
   return (
-    <AnimatedMenuContainer onScreen={onScreen}>
-      <MenuContainer>
-        <Option {...harpKeyOptionProps} />
-        <Option {...pozitionOptionProps} />
-        <Option {...rootPitchOptionProps} />
-        <Option {...displayModeOptionProps} />
-      </MenuContainer>
-    </AnimatedMenuContainer>
+    <Animated.View
```

Comment:
This entire object could potentially be spun out as a wrapper which just takes the Option components as children. That would reduce the duplication between this and the other menu.

Reply:
NIT recorded in open issue to be addressed later: https://github.com/js-jslog/harpnative/issues/56#issuecomment-673620700